.gitignore

```diff
@@ -9,6 +9,7 @@
 /log/*.log
 /tmp
 /.sass-cache
+/.byebug_history
```

Comment:
Yeah I was wondering why that was still there, turns out I still have to delete it, even though it's been ignored. Future versions of this file that get created won't make it to the posted code. :+1: